client/src/Components/Media/Media.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { Media } from "reactstrap";
+
+import usersData from "../../Data/profileData.json";
+
+var notesList = usersData.notes;
+
+import "./Media.css";
+
+const Example = () => {
+  return (
+    <div className="container">
+      <div className="row">
+        {notesList.map((post) => {
+          return (
+            <div key={post.id} className="col-12">
+              <Media>
+                {/* <Media left href="#">
+              <Media
+                object
+                data-src="holder.js/64x64"
+                alt="Generic placeholder image"
+              />
+            </Media> */}
+                <Media body>
+                  <Media heading>
+                    {post.topic} . <small>{post.subject}</small>
+                  </Media>
+                  {post.description}
+                  <br />
+                  <a className="view-more" href="#">
+                    {" "}
+                    View More{" "}
+                    <span className="fas fa-arrow-right fa-lg"> </span>
```

Comment:
These Font Awesome Icons don't seem to work here anywhere. Can you share some snaps off your Local Development to show if they are working? (In my case, they aren't)